Contributing/SwiftStyleGuide.swift

```diff
@@ -119,7 +119,7 @@ enum MyCustomError: Error {
 private extension MyStruct {
 
     func somethingPrivate() -> String {
-        return Bool.random() ? .saturn : .nepturn
+        return Bool.random() ? .saturn : .neptune
```

Comment:
I realize now that this may have been humorous, but I didn't get that when reading it as a newcomer 😆

Reply:
sat**urn** -> nept**urn**